pkg/faq/faq.go

```diff
@@ -364,13 +366,43 @@ func detectFormat(file File) (formats.Encoding, error) {
 	if err != nil {
 		return nil, err
 	}
-	format := linguist.LanguageByContents(fileBytes, linguist.LanguageHints(file.Path()))
+
+	hints := linguist.LanguageHints(file.Path())
+	format := linguist.Analyse(fileBytes, hints)
```

Comment:
just one line it, bro

```go
format := linguist.Analyse(fileBytes, linguist.LanguageHints(file.Path()))
```

Reply:
I had the old code before which used the hint variable, but got rid of it so hadn't gotten to that

Reply:
Fixed